CRM/Core/ClassLoader.php

```diff
@@ -181,12 +181,7 @@ private function getHtmlPurifierPath() {
     // we do this to prevent a autoloader errors with joomla / 3rd party packages
     // Use absolute path, since we don't know the content of include_path yet.
     // CRM-11304
-    if (isset($GLOBALS['civicrm_paths']['civicrm.packages']['path'])) {
-      $file = rtrim($GLOBALS['civicrm_paths']['civicrm.packages']['path'], DIRECTORY_SEPARATOR) . '/IDS/vendors/htmlpurifier/HTMLPurifier/Bootstrap.php';
-    }
-    else {
-      $file = dirname(__FILE__) . '/../../packages/IDS/vendors/htmlpurifier/HTMLPurifier/Bootstrap.php';
-    }
+    $file = dirname(__FILE__) . '/../../vendor/ezyang/htmlpurifier/library/HTMLPurifier/Bootstrap.php';
```

Comment:
I'm wondering why we do this at all. Why not just rely on the composer autoloader?